### DISCUSSION TITLE
[#1065640] Weird snapping behaviour with pasting

### DISCUSSION THREAD

**Discussion by FrederoxDev** [Mojang/minecraft-editor#125] [Bugs]
Video was too big to attach, here it is on the #editor-mode discord channel https://discord.com/channels/523663022053392405/1084090299120373760/1121788880761405461

I copied the selection and went to paste. It did keep snapping back to the original volumes coordinates when I entered a value and pressed enter to confirm the value

**Comment by minecraftchloe** [comment]
Thank you!!
